bin/node/runtime/src/lib.rs

```diff
@@ -131,6 +132,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
 	transaction_version: 2,
+	state_version: 1,
```

Comment:
[Version of the state implementation used by this runtime](https://github.com/paritytech/substrate/pull/9732)